src/UI/templates/default/Input/tpl.tag_input.html

```diff
@@ -1,5 +1,5 @@
 <div id="container-{ID}" class="form-control form-control-sm il-input-tag">
-	<input type="text" id="{ID}" value="{VALUE_COMMA_SEPARATED}" class="form-control form-control-sm"/> <input type="hidden" id="template-{ID}" value='{NAME}[]'>
+	<input type="text" id="{ID}" value="{VALUE_COMMA_SEPARATED}"<!-- BEGIN disabled --> {DISABLED}<!-- END disabled --> class="form-control form-control-sm"/> <input type="hidden" id="template-{ID}" value='{NAME}[]'>
```

Comment:
Maybe this is not enough here. Technically, the tag-input will be disabled, so that the user cannot modify it. But visually, it still looks enabled because of the <div> over it. The user can also click on it and on the first click, even a pointer comes up.
My idea would be to additionally extend the <div> with a "disabled" class. Adding some css for the color and pointer-event would make the division visually disabled for the user.